.github/workflows/link-check-all.yml

```diff
@@ -0,0 +1,29 @@
+name: Link Check All
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  run:
+    name: Link Check All Job
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Run the link check script
+      id: check
+      uses: "iterative/link-check.action@v0.3.0"
+      with:
+        source: "filesystem"
+        configFile: "./.linkcheckrc.json"
+
+    - uses: LouisBrunner/checks-action@v0.1.0
```

Comment:
curious - do we need this for the cron-style check? what would it give us?

Reply:
Good point that this invocation doesn't necessarily need to be a GitHub Check, but currently we need it as the GHA script always makes output for `checks-action`.
That said, it'd be pretty simple for me to add an option to the Action inputs for using the console instead of GHA outputs for runs that don't need to be a Check if that's desirable.